torch/library.h

```diff
@@ -603,6 +604,19 @@ class TORCH_API Library final {
     c10::FunctionSchema s = schema(std::forward<Schema>(raw_schema));
     return _def(std::move(s), nullptr, tags, rv);
   }
+
+  /// Specifies a Python import to tie this library to. When loading
+  /// the library via torch.ops.load_library("lib.so") from Python,
+  /// it will automatically import the Python module (if it has not already
+  /// been imported). This is useful for separating the custom operator
+  /// definitions between a Python module and a C++ shared library.
+  ///
+  /// See NOTE: [Requiring Python imports with TORCH_LIBRARY] for more details.
+  Library& require_pyimport(std::string pyimport) {
```

Comment:
Why not a const char*?

Reply:
It is unclear to me when to prefer std::string vs const char* while programming C++